src/_guides/language/language-tour.md

```diff
@@ -1723,32 +1723,38 @@ assert(foo() == null);
 
 ## Operators
 
-Dart supports the operators shown in the following table.
+Dart supports the operators shown in the following table,
+which **approximates** Dart's operator associativity 
+as well as operator precedence from highest to lowest.
```

Comment:
**comment:** I like where @eernstg is going with his comment. I would recommend more shorter sentences than longer sentences.

Reply:
Thanks for the feedback! I had some trouble incorporating the full "source of truth" portion in to the primary text as many readers may not be familiar with what a "language grammar" is and I'm worried it may scare them away from this section. I added a little elaboration to the primary text and added the "source of truth" text to the warning which is rather prominent. How does that seem?

As for the example and explanation of precedence, we have one right after the table. I added a link to there instead. We don't mention the underlying grammar, but I think that is probably good as many readers of this may not know how to read it, and it could just confuse them. We also don't want too much text/description in the language tour.

Reply:
Sounds good. You are probably right that it is unreasonable to try to describe any subtle details here.